juniper_official/routing/collect-isis-counters.rule

```diff
@@ -0,0 +1,91 @@
+                    frequency 60s;
+                }
+            }			
+            field instance-name {
+                sensor isis-oc {
+                    where "/network-instances/network-instance/@name =~ /{{inst-name}}/";
+                    path "/network-instances/network-instance/@name";
+                }			
+                type string;
+                description "instance-name from the lsdb path";
+            }
+            field protocol-name {
+                sensor isis-oc {
+                    where "/network-instances/network-instance/protocols/protocol/@identifier=~/{{proto-name}}/";
+                    path "/network-instances/network-instance/protocols/protocol/@identifier";
+                }			
+                type string;
+                description "protocol-name from the lsdb path";
+            }
+            field level-num {
+                sensor isis-oc {
+                    where "/network-instances/network-instance/protocols/protocol/isis/levels/level/@level-number=~/{{level-numb}}/";
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/@level-number";
+                }
+                type string;
+                description "level-number from the lsdb path";
+            }
+            field spf-runs {
+                sensor isis-oc {
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/system-level-counters/state/spf-runs";
+                }
+                type string;
+                description "lsp-install-time from the lsdb path";
+            }
+            field seq-num-skips {
+                sensor isis-oc {
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/system-level-counters/state/seq-num-skips";
+                }
+                type string;
+                description "lsp-sequence-number from the lsdb path";
+            }
+            variable inst-name {
+                value .*;
+                description "Enter a regular expression to match the instance name, e.g., 'master|RI1'";
+                type string;
+            }
+            variable proto-name {
+                value .*;
+                description "Enter a regular expression to match the protocol name, e.g., 'isis' or 'ospf'";
+                type string;
+            }
+            variable level-numb {
+                value .*;
+                description "Enter a regular expression to match the level number, e.g., '1', '2', or '3'";
+                type string;
+            }
+            rule-properties {
+                version 1;
+                contributor juniper;
+                supported-healthbot-version 4.3.0;
+                supported-devices {
+                    juniper {
+                        operating-system junos {
+                            products MX {
```

Comment:
Are we supporting only MX960. Please make sure all the models are added here

Reply:
Currently supporting only for MX960 and will be adding other platforms in testing phase.

Reply:
Can you confirm that RB will not be instantiating this rule for any other models?